runtime/jcl/common/system.c

```diff
@@ -477,13 +477,22 @@ jobject getPropertyList(JNIEnv *env)
 		javaVM->directByteBufferMemoryMax = heapSize - (heapSize / 8);
 	}
 #endif /* defined(OPENJ9_BUILD) && JAVA_SPEC_VERSION == 8 */
+#if !defined(OPENJ9_BUILD)
+	/* Don't set a default value for IBM Java 8 */
 	if ((UDATA) -1 != javaVM->directByteBufferMemoryMax) {
+#endif /* !defined(OPENJ9_BUILD) */
```

Comment:
Please finish the sentence with a period.

My preference would be to move `{` onto a line by itself to avoid the need for the conditional `}` below.

Reply:
Updated.